frontend/elements/README.md

```diff
@@ -120,6 +120,7 @@ const defaultOptions = {
                                    // from displaying any translations.
     translationsLocation: "/i18n", // The URL or path where the translation files are located.
     fallbackLanguage: "en",        // The fallback language to be used if a translation is not available.
+    cookieName: "hanko"            // The name of the cookie the session token is stored in.
```

Comment:
Hmm shouldn't it be the `storageKey`? Because it controls both parameters, the `cookieName` and the local storage key. Which leads me to the next question. Shall we rename `storageKey` to `localStorageKey`, where it is really the key used for storing into the local storage and whenever it means both parameters, we call it `storageKey`?

Reply:
Yes, looks like a good idea.